lib/server.js

```diff
@@ -26,14 +26,9 @@ class OAuth2Server {
 
   /**
  * Authenticate a token.
- * Note, that callback will soon be deprecated!
  */
 
   authenticate (request, response, options) {
-    if (typeof options === 'string') {
-      options = {scope: options};
-    }
-
```

Comment:
I removed this completely since it was not supporting al allowed scope input values. This was also not clearly documented